src/main/kotlin/no/nav/syfo/service/FellesformatMapperService.kt

```diff
@@ -113,9 +115,9 @@ fun mapsmRegisteringManuelltTilFellesformat(
                     }
                     content = XMLRefDoc.Content().apply {
                         any.add(HelseOpplysningerArbeidsuforhet().apply {
-                            syketilfelleStartDato = smRegisteringManuellt.syketilfelleStartDato
+                            syketilfelleStartDato = smRegisteringManuell.syketilfelleStartDato
                             pasient = HelseOpplysningerArbeidsuforhet.Pasient().apply {
-                                navn = NavnType().apply {
+                                navn = NavnType().apply {// TODO: Denne må fylles ut. Hentes fra PLD?
```

Comment:
Ja, det kan hentes fra PDL :) Det enkleste er kanskje å lage et Person-objekt som inneholder navn og fnr og sende det inn hit, så slipper du tusen inputparametre :P

Reply:
Edit: Ser at du tar inn PdlPerson, det kan du gjøre for pasienten også. Men, jeg ville kanskje laget et eget internt Person-objekt sånn at vi slipper å endre overalt hvis PdlPerson-grensesnittet endres av en eller annen grunn :) Men akkurat PDL er ikke den verste i så måte, der lager jo vi objektet selv basert på hva vi har behov for, så det er kanskje greit sånn...

Reply:
Smart!